internal/k8s/ako_init.go

```diff
@@ -53,7 +53,7 @@ func PopulateCache() error {
 	avi_obj_cache := avicache.SharedAviObjCache()
 	// Randomly pickup a client.
 	if avi_rest_client_pool != nil && len(avi_rest_client_pool.AviClient) > 0 {
-		_, _, err := avi_obj_cache.AviObjCachePopulate(avi_rest_client_pool.AviClient[0], utils.CtrlVersion, utils.CloudName)
+		_, _, err := avi_obj_cache.AviObjCachePopulate(avi_rest_client_pool.AviClient, utils.CtrlVersion, utils.CloudName)
```

Comment:
Isn't AviClient a list anymore ? Has the structure of AviRestClientPool changed ?

Reply:
It is the same slice. Instead of passing the first element, now I am passing the whole slice so that the called function can choose to use various indices to parallelise operations.